Bottom: a84f3a159dd9126f12ef6854a797b9e0dcdbf321
Top:    06853a301391158299c28be6011861e4de091813
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-22 14:18:49 -0700

Refresh of dbg-fork

---

diff --git a/src/preload.c b/src/preload.c
index cdca9df..c96d1d7 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -413,8 +413,8 @@ int socket(int domain, int type, int protocol)
 		return index;
 
 	recursive = 1;
-//	ret = rsocket(domain, type, protocol);
 goto realsock;
+	ret = rsocket(domain, type, protocol);
 	recursive = 0;
 	if (ret >= 0) {
 		if (fork_support) {
